package.json

```diff
@@ -4,6 +4,7 @@
   "scripts": {
     "build": "vite build",
     "dev": "vite",
+    "federalist": "npm run build -- --base=$BASEURL",
     "prettier:fix": "prettier --write src/",
```

Comment:
I didn't notice 'til now, but with Prettier integrated into ESLint, we shouldn't need a separate `test:prettier`, since that'll be included in the lint. Also, a `prettier:fix` could be implemented as `npm run test:lint -- --fix`.

Reply:
neat, updated in 3a390a4a42321ea192d31a3980665faf25a7f244